sci-physics/vgm/vgm-4.4.ebuild

```diff
@@ -29,6 +29,14 @@ DEPEND="${RDEPEND}
 	doc? ( app-doc/doxygen[dot] )
 	test? ( geant4? ( xml? ( >=sci-physics/geant-4.10.03[gdml] ) ) )"
```

Comment:
The `xml` use flag doesn't make sense. We're not adding XML capabilities to the package, so it's better to just drop `xml`.